### DISCUSSION TITLE
readding CreatePlus because bug fixed

### DISCUSSION THREAD

**Discussion by Robocraft999** [NillerMedDild/Enigmatica6#1946] [General]
Hello im the Programmer of CreatePlus,

I fixed the problem https://github.com/NillerMedDild/Enigmatica6/issues/1720 in the version v3.1.2 and want to ask you if you could re-add the mod, because the main feature of the mod are the goggle-helmets not the curio slot.

Thank you
Robocraft999

**Comment by NielsPilgaard** [comment]
Hi there! Create Plus was re-added in v0.4.8 :)
